.ci/packer_cache.sh

```diff
@@ -0,0 +1,14 @@
+# shellcheck disable=SC1091
+source /usr/local/bin/bash_standard_lib.sh
+
+readonly NODEJS_VERSION="12"
```

Comment:
Is this version defined somewhere else that could be evaluated? Then we don't need to worry about this file if it changes in the future

Reply:
Yeah, we could add it to a static file and reuse it. I like the idea.